src/ORM/DB.php

```diff
@@ -21,6 +22,22 @@
  */
 class DB
 {
+    /**
+     * A dynamic connection that will use either a replica connection if one is
+     * available and not forced to use the 'primary' connection, or the 'primary' connection
```

Comment:
```suggestion
     * A dynamic connection that will use either a replica connection (if one is
     * available and not forced to use the 'primary' connection), or the 'primary' connection
```
Added parenthesis because without it the final "or the 'primary' connection" feels like it's just accidental repetition of "not forced to use the 'primary' connection"

Reply:
Updated